contracts/strategies/VariableRewardsStrategy.sol

```diff
@@ -285,9 +310,7 @@ abstract contract VariableRewardsStrategy is YakStrategyV2 {
             "VariableRewardsStrategy::Emergency withdraw minimum return amount not reached"
         );
         emit Reinvest(totalDeposits(), totalSupply);
-        if (DEPOSITS_ENABLED == true && _disableDeposits == true) {
-            updateDepositsEnabled(false);
-        }
+        updateDepositsEnabled(false);
```

Comment:
This needs to be within an if statement as it will revert if deposits are already disabled


    function updateDepositsEnabled(bool newValue) public onlyOwner {
        require(DEPOSITS_ENABLED != newValue);
        DEPOSITS_ENABLED = newValue;
        emit DepositsEnabled(newValue);
    }